.github/workflows/update-dependencies.yml

```
@@ -0,0 +1,31 @@
+name: Update dependencies
+
+on: workflow_dispatch
+
+jobs:
+  update-dependencies:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      name: Install Python 3.9
+      with:
+        python-version: 3.9
+        architecture: x64
+    - name: Install dependencies
+      run: python -m pip install requests pip-tools
+    - name: Run update
+      run: python ./bin/update_dependencies.py
+    - name: Create Pull Request
+      if: github.ref == 'refs/heads/master' && always()
```

this might be some debug code left over?
```suggestion
      if: github.ref == 'refs/heads/master'
```

---

`always()` refers to the previous step - otherwise, there is an implicit `&& success()` here, IIRC. Not sure if you do want this to make a PR if the script fails, though?

---

The `aways()` is a leftover of a repo where I want to create the PR even if the previous step failed. we don't want to use `always()` here. good catch.